custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt

```diff
@@ -5,94 +5,114 @@ import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
 import jces1209.vu.MeasureType
 import jces1209.vu.memory.SeededMemory
 import jces1209.vu.page.JiraTips
-import jces1209.vu.page.boards.sprint.SprintPage
-import jces1209.vu.page.boards.view.ScrumBoardPage
-import jces1209.vu.page.boards.view.SprintBoardComponent
+import jces1209.vu.page.boards.view.ScrumBacklogPage
+import jces1209.vu.page.boards.view.ScrumSprintPage
 import org.apache.logging.log4j.LogManager
 import org.apache.logging.log4j.Logger
 
 class WorkOnSprint(
     private val meter: ActionMeter,
     private val jiraTips: JiraTips,
-    private val scrumBoardsMemory: SeededMemory<ScrumBoardPage>
+    private val backlogsMemory: SeededMemory<ScrumBacklogPage>,
+    private val sprintsMemory: SeededMemory<ScrumSprintPage>
 ) : Action {
     private val logger: Logger = LogManager.getLogger(this::class.java)
 
     override fun run() {
-        val scrumBoard = scrumBoardsMemory.recall()
-        if (scrumBoard == null) {
+        workOnBacklog()
+

```

Comment:
formatting

Reply:
Do you mean the empty line? I like this one here - divider of different actions